diff --git a/external/d3q b/external/d3q
--- a/external/d3q
+++ b/external/d3q
@@ -1 +1 @@
-Subproject commit 6e5f97816af439702196c248ec2ad0092c561b89
+Subproject commit 6e5f97816af439702196c248ec2ad0092c561b89-dirty
diff --git a/external/devxlib b/external/devxlib
--- a/external/devxlib
+++ b/external/devxlib
@@ -1 +1 @@
-Subproject commit a6b89ef77b1ceda48e967921f1f5488d2df9226d
+Subproject commit a6b89ef77b1ceda48e967921f1f5488d2df9226d-dirty
